test/lsp.py

```diff
@@ -118,6 +214,10 @@ def send_notification(self, name: str, params: Optional[dict] = None) -> None:
 SGR_STATUS_FAIL = '\033[1;31m'
```

Comment:
Interesting! I only knew those as ANSI escape sequences, so I thought `ANSI` as prefix would be better. But after checking what `SGR` really means, it seem to be more accurate :)

Reply:
@aarlt while these are ANSI VT sequences, the specific term here is actually SGR for "Select Graphics Rendition", as these VT sequences are to change the graphics rendition of the painting ANSI text cursor. :-)